### PR TITLE
style(modeling): small changes from deep dives into the BSP trees

### PR DESCRIPTION
**Pull request by z3dev**
There are changes from my deep dives into the BSP trees. Most of the changes are just style changes, but there are a few small logic changes. Hopefully, these make life easier for the next developer.

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [X] Does your submission pass tests?

### CODE REVIEW

File: packages/modeling/src/operations/booleans/trees/splitPolygonByPlane.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const splitPolygonByPlane = (splane, polygon) => {
     const MINEPS = -EPS
     for (let i = 0; i < numvertices; i++) {
       const t = vec3.dot(splane, vertices[i]) - splane[3]
-      const isback = (t < 0)
+      const isback = (t < MINEPS)
```
Comment (platypii) [inline packages/modeling/src/operations/booleans/trees/splitPolygonByPlane.js:39]: I had to think for a minute about why this change. It guess it needs to match line 42 though.

**Review by platypii — APPROVED** [review]
Looks good. Ran the tests, and viewed a couple models. No problems seen in my tests.

**Review by hrgdavor — APPROVED** [review]
looks good :)
